apps/studio/components/interfaces/SQLEditor/DownloadSnippetModal.tsx

```diff
@@ -141,7 +120,43 @@ const DownloadSnippetModal = ({ id, ...props }: DownloadSnippetModalProps) => {
                 </CodeBlock>
               )}
             </Tabs.Panel>
+            <p className="text-xs text-lighter mt-4 text-right mr-6 mb-4">
+              * Run from your local <code>/supabase</code> directory
```

Comment:
This should be project root right? Otherwise we should update each command to remove the `./supabase` prefix.